aip/general/0145.md

```diff
@@ -48,6 +48,25 @@ Exclusive end values are preferable for the following reasons:
 - It is easier to reason about abutting ranges: `[0, x), [x, y), [y, z)`, where
   values are chainable from one range to the next.
 
+### Use of `google.type.Interval`
+
+The [`google.type.Interval`][interval] type (in the common protos described in
```

Comment:
Nit: I think this paragraph could read better as,

> The `google.type.Interval` type, found amongst the common protos that are described in AIP-213, represents a range between two timestamps. The lower bound is defined by an *inclusive* start value and the upper bound by an *exclusive* end value.

, which is similar to the verbiage in the comment above. You *could* pick one based on whether or not you decide to go with the recommendation to change the section header.